src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -174,6 +180,68 @@ private static void StopBridge(CommandLineArguments commandLineArgs)
             BridgeController.StopBridgeProcess(0);
         }
 
+        // Asks the Bridge to release all its resources but continue running
+        private static void ResetBridge(CommandLineArguments commandLineArgs)
+        {
+            string errorMessage = null;
+
+            if (!PingBridge(commandLineArgs.BridgeConfiguration.BridgeHost,
+                                           commandLineArgs.BridgeConfiguration.BridgePort,
+                                           out errorMessage))
+            {
+                Console.WriteLine("The Bridge is not running: {0}", errorMessage);
+                Environment.Exit(0);
+            }
+
+            string bridgeUrl = String.Format("http://{0}:{1}/Resource", commandLineArgs.BridgeConfiguration.BridgeHost, commandLineArgs.BridgeConfiguration.BridgePort);
+            string problem = null;
+
+            Console.WriteLine("Resetting the Bridge by sending DELETE request to {0}", bridgeUrl);
+
+            // We reset the Bridge using a DELETE request to the /resource endpoint.
+            using (HttpClient httpClient = new HttpClient())
+            {
+                try
+                {
+                    var response = httpClient.DeleteAsync(bridgeUrl).GetAwaiter().GetResult();
+                    if (!response.IsSuccessStatusCode)
+                    {
+                        problem = String.Format("{0}Bridge returned unexpected status code='{1}', reason='{2}'",
+                                                    Environment.NewLine, response.StatusCode, response.ReasonPhrase);
+                        if (response.Content != null)
+                        {
+                            string contentAsString = response.Content.ReadAsStringAsync().GetAwaiter().GetResult();
+                            if (contentAsString.Length > 1000)
+                            {
+                                contentAsString = contentAsString.Substring(0, 999) + "...";
+                            }
```

Comment:
Yes, I think we want the Bridge to capture both its own logs as well as access the WCF service's traces.  But that's another feature for another day.  I'll open an issue to track it.